src/Widget/Widget.php

```diff
@@ -241,19 +190,25 @@ public function renderFile(string $file, array $params = []): string
      * The default implementation returns the 'views' subdirectory under the directory containing the widget class file.
      *
      * @return string the directory containing the view files for this widget.
+     *
+     * @throws \InvalidArgumentException
      */
     public function getViewPath(): string
     {
         $class = new ReflectionClass($this);
 
+        if (!is_string($class->getFileName())) {
```

Comment:
Can `getFileName()` return non-string?

Reply:
If it was a suggestion of phpstan.

Reply:
It seems to be wrong: https://www.php.net/manual/en/reflectionclass.getfilename.php

Reply:
It potentially can return false but it would never happen in our case.

Reply:
Let's remove this check. It's not necessary/

Reply:
Done.